api/src/routing.js

```diff
@@ -194,6 +194,13 @@ app.all('/+admin(/*)?', (req, res, next) => {
   next();
 });
 
+app.all('/+medic-user-[^ ]+-[^ ]+(/*)?', (req, res, next) => {
```

Comment:
I think something like this would be easier to read: `/+medic-user-*-meta(/*)?` . It will also resemble the `metaPathPrefix` we define lower in the file. Maybe this route could be moved there, so all rules are grouped together.